### PR TITLE
Add Dependabot explanation

### LINKED ISSUE #41: Add explanation on dependabot for gh action version update

### PR DESCRIPTION
**Pull request by amanmdesai**
Closes #41

### CODE REVIEW

**Comment by klieret** [comment]
Thanks for starting this!

We should add links to the dependabot docs and an example of the config for github actions, e.g. [mine](https://github.com/klieret/python-cookiecutter/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/.github/dependabot.yml) and tell them where to place it

**Comment by amanmdesai** [comment]
Hi thanks. 

I have updated the file. Could you please check again?

File: _episodes/08-adding-ci-to-your-code.md [hunk 1]
```diff
@@ -284,7 +284,23 @@ Great, so we finally got it working... Let's build both the version of the code
 
 Github actions are accompanied by the tags ("@v2"...) which are versions/tags of that action. One might need to update this tags for example from "@v2" to "@v3" because the Github actions developers may fix existing bugs to the action or there may be other updates.
 
-However, this process can be automated by using "Dependabot" which ensures that the workflow references the updated version of the action. If that is not the case, the Dependabot will open a pull request updating the tag of the github action.
+However, this process can be automated by using "Dependabot" which ensures that the workflow references the updated version of the action. If that is not the case, the Dependabot will open a pull request updating the tag of the Github action.
 
+The dependabot action can be added to a Github repository by creating file `dependabot.yml` in the `.github/` folder. The content of the file looks like this:
```
Comment (klieret) [inline _episodes/08-adding-ci-to-your-code.md]: the file

File: _episodes/08-adding-ci-to-your-code.md [hunk 2]
```diff
@@ -284,7 +284,23 @@ Great, so we finally got it working... Let's build both the version of the code
...
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+ ~~~
+{: .language-yaml}
+
+where interval is the frequency of looking for updates to Github actions.
+
+For more information on Dependabot, see e.g., [LINK](https://docs.github.com/en/code-security/dependabot)
```
Comment (klieret) [inline _episodes/08-adding-ci-to-your-code.md]: LINK -> here?

Comment (klieret) [reply _episodes/08-adding-ci-to-your-code.md]: also missing full stop

**Review by klieret — CHANGES_REQUESTED** [review]

File: _episodes/08-adding-ci-to-your-code.md (hunk 1 above)
Comment (amanmdesai) [reply _episodes/08-adding-ci-to-your-code.md]: done

File: _episodes/08-adding-ci-to-your-code.md (hunk 2 above)
Comment (amanmdesai) [reply _episodes/08-adding-ci-to-your-code.md]: done

**Comment by amanmdesai** [comment]
Hi @klieret Implemented the changes

**Review by klieret — APPROVED** [review]
Thanks!

**Comment by klieret** [comment]
@all-contributors please add @amanmdesai  for content

**Pull request merged**